src/lib/Libcmds/batch_status_find.c

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 1994-2018 Altair Engineering, Inc.
+ * For more information, contact Altair at www.altair.com.
+ *
+ * This file is part of the PBS Professional ("PBS Pro") software.
+ *
+ * Open Source License Information:
+ *
+ * PBS Pro is free software. You can redistribute it and/or modify it under the
+ * terms of the GNU Affero General Public License as published by the Free
+ * Software Foundation, either version 3 of the License, or (at your option) any
+ * later version.
+ *
+ * PBS Pro is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Commercial License Information:
+ *
+ * For a copy of the commercial license terms and conditions,
+ * go to: (http://www.pbspro.com/UserArea/agreement.html)
+ * or contact the Altair Legal Department.
+ *
+ * Altair’s dual-license business model allows companies, individuals, and
+ * organizations to create proprietary derivative works of PBS Pro and
+ * distribute them - whether embedded or bundled with other software -
+ * under a commercial license agreement.
+ *
+ * Use of Altair’s trademarks, including but not limited to "PBS™",
+ * "PBS Professional®", and "PBS Pro™" and Altair’s logos is subject to Altair's
+ * trademark licensing policies.
+ *
+ */
+
+#include <stdio.h>
+#include <string.h>
+
+#include "pbs_ifl.h"
+
+/**
+ * @file	batch_status_find.c
+ */
+
+/**
+ * @brief
+ *	find_batch_status - find a batch_status with given name in a batch_status structures list.
+ *
+ * @param[in] bs - batch_status linked list
+ * @param[in] name - name of the batch_status structure to be searched
+ *
+ *
+ * @return 	batch_status structure handle
+ * @retval	batch_status structure pointer with given name or NULL
+ *
+ */
+struct batch_status *find_batch_status(struct batch_status *bs, const char *name) {
+
+	if (name == NULL)
+		return NULL;
+
+	for( ; ((bs != NULL) && strcmp(name, bs->name)); bs = bs->next )
```

Comment:
Another style nit: remove the spaces after '(' and before ')', see the first bullet under "Use of White Space" section of the coding standards.

Reply:
Ok will delete it.